packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,205 @@
+import { Badge } from 'terra-clinical-item-view/package.json?dev-site-package';
+import { Notice } from "@cerner/terra-docs";
+import ItemViewWithHeader from '../example/ItemViewWithHeader?dev-site-example';
+import ItemViewTwoColumnWithHeader from '../example/ItemViewTwoColumnWithHeader?dev-site-example';
+
+<Badge />
+
+# Accessibility Guide for Terra Clinical Item View
+
+## Why is this important?
+
+Terra Clinical Item View allows you to create a view filled with information in the form of Clinical Item Displays. Improper usage of this component may prevent end users from understanding content and necessary context to interact with the page. 
+
+## Accessibility Considerations
+
+#### General notes
+- **Related information should be presented as close to each other as possible.** It is best practice to keep related information above or below its related counterparts.
+  - For example - presenting related information across from each other in the two column view can make it more difficult for users to understand that the information is related.
+Reading by column from top to bottom is the more common way information is consumed visually, so when presenting this information by row between the two columns it might be confusing for assistive technology users and users in general.
+- When passing in Clinical Item Displays, keep in mind that only 8 can be used within a view at a time. Any additional displays passed in will be ignored.
+- Displays are also split between two columns with every other display being moved to the right hand column.
```

Comment:
Slightly reworded this and added the info to the layout prop since I think that's more relevant than putting it under the display prop. [facd8ef](https://github.com/cerner/terra-clinical/pull/916/commits/facd8ef13234010bbe60a865507a585fef7b3a20)